libs/test/SmartStoreTest/src/com/salesforce/androidsdk/store/SqliteLibraryLoaderTest.java

```diff
@@ -0,0 +1,61 @@
+/*
+ * Copyright 2015 Salesforce.com.
+ * All Rights Reserved.
+ * Company Confidential.
+ */
+package com.salesforce.androidsdk.store;
+
+import java.io.File;
+
+import org.mockito.Mockito;
+
+import com.salesforce.androidsdk.smartstore.store.SqliteLibraryLoader;
+
+import android.content.Context;
+import android.test.AndroidTestCase;
+
+import static org.mockito.Matchers.any;
```

Comment:
We don't use `Mockito` and don't want to add this dependency. So this test will have to be rewritten as a pure `JUnit` test, or has to be removed.